WordPress/Classes/Networking/ServiceRemoteREST.m

```diff
@@ -1,6 +1,10 @@
 #import "ServiceRemoteREST.h"
 #import "WordPressComApi.h"
 
+static NSString* const ServiceRemoteRESTApiVersionStringInvalid = @"invalid_api_version";
+static NSString* const ServiceRemoteRESTApiVersionString_1_1 = @"v1.1";
+static NSString* const ServiceRemoteRESTApiVersionString_1_2 = @"v1.2";
```

Comment:
Even though it's static (thus locally scoped), we can still have name-clashes with other globally-scoped constants.

See this example declaration of a `static const`:
<img width="821" alt="screen shot 2015-08-18 at 3 05 33 pm" src="https://cloud.githubusercontent.com/assets/1836005/9338372/d09ed830-45ba-11e5-903d-9210ae39ca2a.png">

The only difference with a globally-scoped constant is that `static` constants won't clash with other `static` constants.